src/System.Private.Uri/tests/FunctionalTests/IriEncodingDecodingTests.cs

```diff
@@ -19,7 +19,11 @@ public class IriEncodingDecodingTest
             { "%2D%5F%2E%7E", @"-_.~" }, // Encoded RFC 3986 Unreserved Marks.
             { "%2F%3F%3A%40%23%5B%5D", "%2F%3F%3A%40%23%5B%5D" }, // Encoded RFC 3986 Gen Delims.
             { @";&=+$,!'()*", @";&=+$,!'()*" }, // RFC 3986 Sub Delims.
-            { "%3B%26%3D%2B%24%2C%21%27%28%29%2A", "%3B%26%3D%2B%24%2C%21%27%28%29%2A" } // Encoded RFC3986 Sub Delims.
+            { "%3B%26%3D%2B%24%2C%21%27%28%29%2A", "%3B%26%3D%2B%24%2C%21%27%28%29%2A" }, // Encoded RFC3986 Sub Delims.
```

Comment:
Will those test data fail on NetFX?

Reply:
They would, but all of these tests come from the reserved character set fix and are already disabled on netfx.

Reply:
Ok.